frontend/src/components/Gallery/Gallery.jsx

```diff
@@ -1,21 +1,35 @@
-import Card from "@components/Cards/Card/Index";
+import { useState, useEffect } from "react";
+import axios from "axios";
+/* eslint-disable import/no-unresolved */
+/* eslint-enable import/no-unresolved */
```

Comment:
... Why disabling a rule and reenabling it just afterwards ? This shouldn't fix anything O_O

Reply:
Nah, delete these two lines and recommit

Reply:
Plz be kind Chief Tribo